src/styles/page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
```

Comment:
Use SCSS variables instead of default CSS variables
https://sass-lang.com/documentation/variables